### PR TITLE
Add specific references to FEC mechanisms

### LINKED ISSUE #73: Figure out what FEC mechanisms should be advertised in offers/answers
Depends on resolution of FEC on mailing list

### PR DESCRIPTION
**Pull request by juberti**
Fixes #73

### CODE REVIEW

File: draft-ietf-rtcweb-jsep.xml [hunk 1]
```diff
@@ -1362,7 +1365,8 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
 
               <t>For each supported codec, "a=rtpmap" and "a=fmtp" lines, as
               specified in <xref target="RFC4566"></xref>, Section 6.
-              The REQUIRED audio and video codecs to support are specified in <xref
+              The mandatory audio and video codecs that MUST be supported are
+              specified in <xref
```
Comment (ekr) [inline draft-ietf-rtcweb-jsep.xml]: mandatory and MUST be supported seem redundant. Maybe remove mandatory?

File: draft-ietf-rtcweb-jsep.xml [hunk 2]
```diff
@@ -1377,8 +1381,12 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
               references the payload type of the primary codec, as specified
               in <xref target="RFC4588"></xref>, Section 8.1.</t>
 
-              <t>For each supported FEC mechanism, a corresponding "a=rtpmap"
-              line indicating the desired FEC codec.</t>
+              <t>For each supported FEC mechanism, "a=rtpmap" and "a=fmtp" lines,
+              as specified in <xref target="RFC4566"></xref>, Section 6. The
+              mandatory FEC mechanisms that MUST be supported are specified in
```
Comment (ekr) [inline draft-ietf-rtcweb-jsep.xml]: mandatory/MUST here as well.

File: draft-ietf-rtcweb-jsep.xml [hunk 3]
```diff
@@ -1437,7 +1445,9 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
               <t>If FEC is supported for this media type, another "a=ssrc"
               line with the FEC SSRC, and an "a=ssrc-group" line, as specified
               in <xref target="RFC5576"></xref>, section 4.2, with semantics
-              set to "FEC" and including the primary and FEC SSRCs.</t>
+              set to "FEC-FR" and including the primary and FEC SSRCs. For
```
Comment (ekr) [inline draft-ietf-rtcweb-jsep.xml]: You need a reference to 5956 for FEC-FR. I have no opinion on whether we want FEC-FR or FEC, but I note that this is a change.

File: draft-ietf-rtcweb-jsep.xml [hunk 4]
```diff
@@ -1836,8 +1846,9 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
               <t>For each supported codec that is present in the offer,
               "a=rtpmap" and "a=fmtp" lines, as specified in <xref
               target="RFC4566"></xref>, Section 6, and <xref
-              target="RFC3264"></xref>, Section 6.1. 
-              The REQUIRED audio and video codecs to support are specified in <xref
+              target="RFC3264"></xref>, Section 6.1.
+              The madatory audio and video codecs that MUST be supported are
+              specified in <xref
```
Comment (ekr) [inline draft-ietf-rtcweb-jsep.xml]: mandatory/MUST again

**Comment by ekr** [comment]
This looks basically good. I see one missing reference and some places where I think the text might read better. Feel free to ignore my advice on the text, but you should probably fix the reference

**Comment by juberti** [comment]
PTAL

**Comment by fluffy** [comment]
LGTM

**Pull request merged**